src/utils/particle/drawer.ts

```diff
@@ -162,7 +176,7 @@ export class Drawer {
         }
     }
 
-    public subDraw(ctx: CanvasRenderingContext2D, img_obj: HTMLImageElement, radius: number) {
+    private subDraw(ctx: CanvasRenderingContext2D, img_obj: HTMLImageElement, radius: number) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'subDraw' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)